packages/flutter/lib/src/material/slider.dart

```diff
@@ -374,6 +381,13 @@ class Slider extends StatefulWidget {
   /// Ignored if this slider is created with [Slider.adaptive]
   final SemanticFormatterCallback semanticFormatterCallback;
 
+  /// Whether to use the updated Material spec version of the [Slider].
```

Comment:
Same comments about this flag as for RangeSlider.

Reply:
Done

Reply:
The first sentence in API doc should be separated from the rest by a blank line.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements